docs/what-is-nuget.md

```diff
@@ -61,7 +61,7 @@ The following image shows a project that depends on five packages, which in turn
 
 Notice that some packages appear multiple times in the dependency graph. For example, there are three different consumers of package B, and each consumer might also specify a different version for that package (not shown). This is a common occurrence, especially for widely-used packages. NuGet fortunately does all the hard work to determine exactly which version of package B satisfies all consumers. NuGet then does the same for all other packages, no matter how deep the dependency graph.
 
-For more details on how NuGet performs this service, see [Dependency resolution](concepts/dependency-resolution.md).
+For more details on how NuGet performs this service, see [Dependency resolution](https://docs.microsoft.com/en-us/nuget/concepts/dependency-resolution).
```

Comment:
This goes against the recommendation here: https://docs.microsoft.com/en-us/contribute/how-to-write-links#links-from-one-article-to-another. 

Please revert this.